tests/TestHelpers/GraphHelper.php

```diff
@@ -532,6 +532,42 @@ public static function addUserToGroup(
 		);
 	}
 
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $adminUser
+	 * @param string $adminPassword
+	 * @param array $userIds
+	 * @param string $groupId
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function addMultipleUsersToGroup(
+		string $baseUrl,
+		string $xRequestId,
+		string $adminUser,
+		string $adminPassword,
+		array $userIds,
+		string $groupId
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'groups/' . $groupId);
+		$body = [ "members@odata.bind" => [] ];
+		foreach ($userIds as $userId) {
+			$body["members@odata.bind"][] = self::getFullUrl($baseUrl, 'users/' . $userId);
+		}
+
+		return HttpRequestHelper::sendRequest(
+			$url,
+			$xRequestId,
+			'PATCH',
+			$adminUser,
+			$adminPassword,
+			self::getRequestHeaders(),
+			\json_encode($body)
+		);
+	}
+
```

Comment:
We have `addUsersToGroup` in `GraphHelper` can't we use or refactor that?

Reply:
I don't think it will be beneficial to reuse that function . We have to create new function to create body where we have to use if else to create different type of body for different request.

Reply:
this function has been removed in past. Don't know why it has been removed.